app/Api/V1/Domain/Problem/Param/ReadProblemParam.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Api\V1\Domain\Problem\Param;
+
+
+class ReadProblemParam
```

Comment:
You should check validation for every input

in this case i think `limit` should be `numeric|min:0|nullable`